sdks/python/apache_beam/runners/common.py

```diff
@@ -558,16 +568,33 @@ def __init__(self, window_fn, main_receivers, tagged_receivers):
     self.main_receivers = main_receivers
     self.tagged_receivers = tagged_receivers
 
+    # TODO: remove if block after per-element-output-counter released
```

Comment:
Please file JIRAs for these TODOs and reference them here (e.g., "TODO(BEAM-1234): ...").

Reply:
I added one JIRA in next line. will delete first line in next revision.